components/tidb_query_vec_expr/src/impl_string.rs

```diff
@@ -1919,4 +1925,37 @@ mod tests {
             assert_eq!(output, expected_output);
         }
     }
+
+    #[test]
+    fn test_lower() {
+        let cases = vec![
+            (Some(b"hello".to_vec()), Some(b"hello".to_vec())),
+            (Some(b"123".to_vec()), Some(b"123".to_vec())),
+            (
+                Some("café".as_bytes().to_vec()),
+                Some("café".as_bytes().to_vec()),
+            ),
+            (
+                Some("数据库".as_bytes().to_vec()),
+                Some("数据库".as_bytes().to_vec()),
+            ),
+            (
+                Some("ночь на окраине москвы".as_bytes().to_vec()),
+                Some("ночь на окраине москвы".as_bytes().to_vec()),
+            ),
+            (
+                Some("قاعدة البيانات".as_bytes().to_vec()),
+                Some("قاعدة البيانات".as_bytes().to_vec()),
+            ),
+            (None, None),
+        ];
+
+        for (arg, exp) in cases {
+            let output = RpnFnScalarEvaluator::new()
+                .push_param(arg.clone())
+                .evaluate(ScalarFuncSig::Upper)
```

Comment:
Why use `Upper`?

Reply:
Sorry. I forgot to modify the test code. Now fixed.

Reply:
> Why use `Upper`?
Sorry. I forgot to modify the test code. Now fixed.